x-pack/platform/plugins/shared/streams/server/lib/tasks/storage.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { types, type IndexStorageSettings, type IStorageClient } from '@kbn/storage-adapter';
+import type { PersistedTask } from './types';
+
+export const taskStorageSettings = {
+  name: '.kibana_streams_tasks',
+  schema: {
+    properties: {
+      id: types.keyword(),
+      type: types.keyword(),
```

Comment:
Is this the `type` of workload, for example sig event/feature/description generation ? Should we create a union type for this ?